MyOwnORM/MyOwnORM/Services/CustomDbSetService.cs

```diff
@@ -0,0 +1,783 @@
+
+            return res.ToArray();
+        }
+        public string GetForeignKeyNameForGenericType()
+        {
+            T type = dbSetReflection.CreateInstanceType();
+            string fkAttribute = dbSetReflection.GetForeignKeyAttribute(type.GetType());
+            string fk = $"{typeof(T).Name}Id";
+            return string.IsNullOrEmpty(fkAttribute) ? fk : fkAttribute;
+        }
+
+        public string GetForeignKeyNameForCustomClass(Type includeType)
+        {
+            string fkAttribute = dbSetReflection.GetForeignKeyAttribute(includeType);
+            string fk = $"{typeof(T).Name}Id";
+            return string.IsNullOrEmpty(fkAttribute) ? fk : fkAttribute;
+        }
+        public string GetForeignKeyNameForIncludeExpressionsMethod(Type includeType, T entity)
+        {
+            string fkAttribute = dbSetReflection.GetForeignKeyAttribute(includeType);
+            string fk = $"{entity}Id";
+            return string.IsNullOrEmpty(fkAttribute) ? fk : fkAttribute;
+        }
+        public string GetSelectQuery(string tableName)
+        {
+            return $"SELECT * FROM {tableName}";
+        }
+        public string GetSelectFilterQuery(string tableName, string key, string val)
+        {
+            return $"SELECT * FROM {tableName} WHERE {key} = {val}";
+        }
+        public string GetInsertIntoQuery(string tableName, string values)
+        {
+            return $"INSERT INTO {tableName} VALUES ({values})";
+        }
+        public string GetUpdateQuery(string tableName, string updateStr, string key, string val)
+        {
+            return $"UPDATE {tableName} SET {updateStr} WHERE {key}={val}";
+        }
+        public string GetDeleteQuery(string tableName, string key, string val)
+        {
+            return $"DELETE FROM {tableName} WHERE {key}={val}";
+        }
+        public async Task GetAllAsyncQuery(string tableName, List<T> entities)
+        {
+            using (SqlConnection connection = new SqlConnection(_connectionString))
+            {
+                connection.Open();
+
+                string query = GetSelectQuery(tableName);
+
+                using (SqlCommand command = new SqlCommand(query, connection))
+                {
+                    using (SqlDataReader reader = command.ExecuteReader())
+                    {
+                        while (reader.Read())
+                        {
+                            T entity = dbSetReflection.MapReaderToEntity(reader);
+                            entities.Add(entity);
+                        }
+                    }
+                }
+            }
+        }
+        public async Task IncludeAsyncQuery(Expression<Func<T, object>> include, string tableName, List<T> entities)
+        {
+            string propVal = GetPropertyValue(include);
+            object includeType = dbSetReflection.GetIncludeType(include);
+
+            using (SqlConnection connection = new SqlConnection(_connectionString))
```

Comment:
You didn't understand me, you separated all this to service and repository is clear, but now your service is like repository before cleaning. Separate opening/closing connection, because now each your method works similar and there is lots of code duplicating